en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -563,50 +548,52 @@ Based on the partisan indicators assigned to the data, blue plot points represen
 
 Note in the following visualization that the colors are not clearly grouped! 
 
-{% include figure.html filename="en-or-text-mining-youtube-comments-09.png" alt="Final Wordfish model visualization, showing the document (comment) scaling components of the model. The plotting symbol used for each document is an open circle. The color of this circle denotes if the document displayed was contributed by a politically left-leaning source (blue circles) or politically right-leaning source (red circles). The horizontal axis (theta) shows the scaling of comments, which is parallel to the horizontal scaling of features. The theta parameter describes which comments are more polarized versus more central, and in which direction. The vertical axis (alpha) reflects a fixed effect for document length - that is, how many features (words) remained in each comment following preprocessing. The distribution of red and blue plotting symbols throughout the model is roughly even." caption="Figure 9. Visualization of WordFish model showing relative comment placement color-coded by partisanship of video channel" %}
+{% include figure.html filename="en-or-text-mining-youtube-comments-09.png" alt="Final Wordfish model visualization, showing the document (comment) scaling components of the model. The plotting symbol used for each document is an open circle. The color of this circle denotes if the document displayed was contributed by a politically left-leaning source (blue circles) or politically right-leaning source (red circles). The horizontal axis (theta) shows the scaling of comments, which is parallel to the horizontal scaling of features. The theta parameter describes which comments are more polarized versus more central, and in which direction. The vertical axis (alpha) reflects a fixed effect for document length - that is, how many features (words) remained in each comment following pre-processing. The distribution of red and blue plotting symbols throughout the model is roughly even." caption="Figure 9. Visualization of WordFish model showing relative comment placement, color-coded by partisanship of video channel" %}

```

Comment:
I noticed that while your new screenshot does now indicate `alpha` instead of `psi` on the y-axis, it still says `beta` on the x-axis. According to the text, I believe this should be `theta`? 

If so, would you kindly send me a new image to my email, publishing.assistant[@]programminghistorian.org? Thank you!

Reply:
I'll check on that and get back to you, thanks for catching it